Library/Formula/docutils.rb

```diff
@@ -0,0 +1,11 @@
+class Docutils < Formula
+  desc "Text processing system for processing plaintext documentation into useful formats"
```

Comment:
Please shorten the description. “`docutils: desc...`” should fit under 80 characters.